elastic/kibana
# [Table list view] Integrate package into visualizations, maps, graphs

In this PR I've integrated the package version of the `<TableListView />` component into the following plugins:

* Visualizations
* Maps
* Graphs

There aren't any new functionality added.

### How to test

* Launch the cloud deployment
* Navigate to the app and make sure the list view renders correctly

### Edit

This PR also update the Table list view to the new `<EuiPageTemplate />` component

<img width="1721" alt="Screenshot 2022-09-08 at 14 59 32" src="https://user-images.githubusercontent.com/2854616/189142566-3a7457e4-6f0b-47da-8d67-442223ee724e.png">
<img width="1710" alt="Screenshot 2022-09-08 at 14 59 48" src="https://user-images.githubusercontent.com/2854616/189142577-bf6667b3-41db-4afc-a887-7a0f9b7a988e.png">
<img width="1724" alt="Screenshot 2022-09-08 at 15 00 03" src="https://user-images.githubusercontent.com/2854616/189142588-0c58efd2-4905-4bf4-bae5-af00f703fb61.png">
<img width="1726" alt="Screenshot 2022-09-08 at 15 00 27" src="https://user-images.githubusercontent.com/2854616/189142592-3e0b6b54-91fc-4672-9f4d-6fdaf5081111.png">
